core/fetcher/fetcher.go

```diff
@@ -142,16 +142,17 @@ func (f *Fetcher) fetchProposerData(ctx context.Context, slot int64, argSet core
 			Slot: slot,
 			Type: core.DutyRandao,
 		}
-		randao, err := f.aggSigDBFunc(ctx, dutyRandao, pubkey)
+		signedData, err := f.aggSigDBFunc(ctx, dutyRandao, pubkey)
 		if err != nil {
 			return nil, err
 		}
-		randaoEth2 := core.DecodeRandaoAggSignedData(randao)
+
+		randao := signedData.Signature().ToETH2()
 
 		// TODO(dhruv): what to do with graffiti?
 		// passing empty graffiti since it is not required in API
 		var graffiti [32]byte
```

Comment:
Let's have graffiti something which includes a string like:
`Charon Distributed Validator`
This shows up on beaconcha.in alongside Public key when our DV proposes a block.

Reply:
sure, another PR though?